web/blueprints/user/__init__.py

```diff
@@ -630,6 +653,24 @@ def list_users_by_traffic_group(traffic_group_id):
                            users=user_list)
 
 
+@bp.route('/<int:user_id>/reset_password', methods=['GET', 'POST'])
+@access.require('user_change')
+def reset_password(user_id):
+    form = UserResetPasswordForm()
```

Comment:
Why wuold a form be needed? Just acting upon GET requests by just doing that would be sufficient.
Implementing some confirmation mechanism can just as well be handled by using a GET parameter `confirm`.

Reply:
I hope @fgrsnau can answer this, as i have not changed this part from his old commit.